src/components/Tagline/Tagline.module.scss

```diff
@@ -0,0 +1,23 @@
+.tagline {
+  padding: 30px 20px;
+  width: 100vw;
+  display: flex;
+  flex-direction: column;
+}
+
+.taglinePara {
+  width: auto;
+}
+
+.paraContainer {
+  display: flex;
+  flex-direction: column;
+}
+
+.h2 {
+  margin-bottom: -30px;
```

Comment:
hmm we shouldn't ever have to do this. this is too hacky. the positioning should just work. if for whatever reason we must do something like this. you should use `position:relative; top:-30px;` to restrict the hack from affecting other layout